crates/sui/src/keytool.rs

```diff
@@ -127,10 +136,13 @@ fn store_and_print_keypair(address: SuiAddress, keypair: SuiKeyPair) {
     let path = Path::new(&path_str);
     let address = format!("{}", address);
     let kp = keypair.encode_base64();
-    let kp = &kp[1..kp.len() - 1];
```

Comment:
dont think this was correct, the flag byte was appended as the first byte before encoding it to base64. skipping the first bytes after encoding does not make sense.